src/test/shared/clients/mockClients.ts

```diff
@@ -538,3 +545,61 @@ export class MockS3Client implements S3Client {
         this.deleteBucket = deleteBucket
     }
 }
+
+export class MockAppRunnerClient implements AppRunnerClient {
```

Comment:
Why not mimic the MockS3Client above? At the very least, you'll want to move to an implementation like that eventually, no?

Reply:
I used mockito so no I don't need it. Half the clients in this file aren't used at all (like the S3 one). We need to be decisive on how we deal with SDK clients, not just do whatever.

Reply:
Do we need MockAppRunnerClient at all?

Reply:
Nope. I just let it auto-generate for me to appease the compiler. Probably better to have the stub for now since it's obvious that it is unused?